scripts/PMID_downloader.py

```diff
@@ -91,29 +91,30 @@ def download_pdf(pmcid: str, pmid: str, pdf_out_dir: str):
 
 @click.command(
     help="""
-Takes a directory to find PMIDs and downloads the corresponding PDFs from PubMed (whenever they are accessible via PubMed Central).
+INPUT: a .pkl file whose entries are strings of the form "PMID_1234567" 
+OUTPUT: a directory containing the corresponding PDFs of the journal articles 
+(whenever they are accessible via PubMed Central).
 
-PMID_FILE_PATH:     Directory where PMID files are located.
-PDF_OUTPUT_DIR:     Ouput directory where PMID PDFs will be written.
+PKL_FILE_PATH:     the file path for the .pkl file
+PDF_OUTPUT_DIR:     where you want the directory containing the PDFs to be located
 
 Example: 
-data/pmid_list.txt      data/pmid_pdfs, 
+data/pmids.pkl      data/pmid_pdfs, 
 """
 )
-@click.argument("pmid_file_path", type=click.Path(exists=True))
+@click.argument("pkl_file_path", type=click.Path(exists=True))
 @click.argument("pdf_out_dir", type=click.Path(exists=False, dir_okay=True))
-@click.option("--recursive_dir_search", type=click.BOOL, default=True)
-def pmid_downloader(pmid_file_path: str, pdf_out_dir: str, recursive_dir_search: bool):
+def pmid_downloader(pkl_file_path: str, pdf_out_dir: str):
     pdf_out_dir_path = Path(pdf_out_dir)
     if not pdf_out_dir_path.exists():
         pdf_out_dir_path.mkdir(exist_ok=True, parents=True)
 
-    pmids = find_pmids(pmid_file_path, recursive=recursive_dir_search)
+    pmids: set = pkl_to_set(pkl_file_path)  # entries of the form "PMID_1234567"
```

Comment:
Think, the comment here is rather a part of the doc string. Otherwise, you would need to mention that everywhere you call this function.